README.md

```diff
@@ -224,14 +224,13 @@ end
 
 ```
 
-Use `elastic_index.load_from_source = true` to configure an index without ActiveRecord.
+Include `ElasticIndex::LoadFromSource` to configure an index without ActiveRecord.
 
 ```ruby
 class Product
   include ActiveModel::Model
   include ElasticRecord::Record
-
-  self.elastic_index.load_from_source = true
+  include ElasticRecord::LoadFromSource
```

Comment:
👎 Use a DSL.

Reply:
Are opposed to the module or how it's included here?

I could add a `loaded_from_source` class method that calls `include`. Then we'd have:

```ruby
class Product
  include ActiveModel::Model
  include ElasticRecord::Record
  loaded_from_source
end
```

Reply:
`elastic_index.load_from_source!`

I like this. It is very similar to the current `elastic_index.load_from_source = true` and the other index configurations we provide via `elastic_index`.

Reply:
Only opposed to adding the module as the way to get the behavior. I am suggesting that `elastic_index.load_from_source!` includes the module.